examples/identity/Forc.toml

```diff
@@ -0,0 +1,7 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "identity"
+
+[dependencies]
```

Comment:
This needs to be a path to the stdlib in this repo, so that that's tested instead of the latest release. Note that the committed `Forc.lock` is using `v0.16.1` instead of this branch's stdlib. See how other examples do this.

Reply:
Should pass now

Reply:
Nevermind, looks like there is a formatting error from an earlier commit suggestion